pkgs/top-level/all-packages.nix

```
@@ -27253,6 +27253,7 @@ with pkgs;
   pybitmessage = callPackage ../applications/networking/instant-messengers/pybitmessage { };
 
   qarma = libsForQt5.callPackage ../tools/misc/qarma { };
+  qarmaForQt6 = libsForQt6.callPackage ../tools/misc/qarma-for-qt6 { };
```

```suggestion
  qarma_qt6 = libsForQt6.callPackage ../tools/misc/qarma/qt6.nix { };
```
I would expect that it would be called like this. Also we can put the file into the same directory as the normal qarma.

---

this is temporary, to compare qt5 and qt6 builds. goal is to move to qt6 by default

---

Thats a huge task. Thanks for trying to do it! If you hit a roadbloack I would be totally fine for having Qt5 and Qt6 alongside each other for a while.